UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.SubscriptionService/SubscriptionServiceJob.cs

```diff
@@ -35,7 +37,7 @@ public async Task ProcessQueueMessage([QueueTrigger("%SubscriptionStorageConfigu
             {
                 try
                 {
-                    await _subscriptionServiceData.CreateOrUpdateSubscription(subscriptionMessage, CancellationToken.None);
+                    eventSubscription = await _subscriptionServiceData.CreateOrUpdateSubscription(subscriptionMessage, CancellationToken.None);
```

Comment:
I'm assuming this is ready for a future PBI.

Reply:
Yes